web/src/test/scala/quasar/api/services/MountServiceSpec.scala

```diff
@@ -80,7 +80,7 @@ class MountServiceSpec extends quasar.QuasarSpecification with ScalaCheck with H
       )
 
       val meff: MEff ~> Task =
-        reflNT[Task] :+: KeyValueStore.fromTaskRef(configsRef)
+        reflNT[Task] :+: KeyValueStore.impl.fromTaskRef(configsRef)
```

Comment:
I am surprised to see this. I made that change locally, but did not think I had opened the PR yet... 😕